static/js/zamboni/devhub.js

```diff
@@ -97,7 +97,7 @@ $(document).ready(function() {
       var $isManualReview = $('#manual-review');
       var $submitAddonProgress = $('.submit-addon-progress');
       function updateListedStatus() {
-        if (!$isUnlisted.is(':checked') || $new_form.data('addon-is-listed')) {  // It's a listed add-on.
+        if (($isUnlisted.length && !$isUnlisted.is(':checked')) || $new_form.data('addon-is-listed')) {  // It's a listed add-on.
```

Comment:
Basically I expected something like 
`if ((!$('#id_is_unlisted').length || !$('#id_is_unlisted').is(':checked')) || $new_form.data('addon-is-listed')) {`